modules/tinymce/src/themes/silver/main/ts/ui/dialog/WindowManager.ts

```diff
@@ -240,6 +240,9 @@ const setup = (extras: WindowManagerSetup): WindowManagerImpl => {
         // 'ResizeWindow` as that's handled by docking already.
         editor.on('ResizeEditor', refreshDocking);
       }
+      editor.on('ScrollWindow', () => {
```

Comment:
I'm a little concerned about maintaining scroll responsiveness. How expensive is reposition popups?